src/source-af-packet.c

```diff
@@ -782,6 +782,7 @@ int AFPReadFromRing(AFPThreadVars *ptv)
             SCReturnInt(AFP_FAILURE);
         }
         PKT_SET_SRC(p, PKT_SRC_WIRE);
+        p->pkt_mode = PKT_MODE_AUTO;
```

Comment:
Exactly, PKT_MODE_AUTO means it can be either IDS or IPS, but I think stream.inline can't be set to AUTO.
Otherwise, another way to set the pkt mode could be looking at p->afp_v.copy_mode.